examples/TargetingConsoleApp/Program.cs

```diff
@@ -63,11 +65,23 @@ public static async Task Main(string[] args)
                         Groups = user.Groups
                     };
 
+                    //
+                    // Evaluate feature flag using targeting
                     bool enabled = await featureManager.IsEnabledAsync(FeatureName, targetingContext, CancellationToken.None);
```

Comment:
For consistency, maybe we should do
IsEnabledAsync<TargetingContext>(....)?

Reply:
Not sure what this applies to. Do you mean to have `IsEnabledAsync` multi-lined like `GetVariantAsync` in this file?

Reply:
Interesting. What is displayed in github is different from what's in the codeflow :).

What I mean is IsEnabledAsync__\<TargetingContext\>__(....)

Reply:
I see. I added it.